frontend/src/app/teams/components/layout/teams-layout.component.html

```diff
@@ -1,4 +1,48 @@
-<ng-container *ngIf="isLoading"> Loading... </ng-container>
+<ng-container *ngIf="isLoading"> 
+	<div class="loadingTop">
```

Comment:
USE NAMESPACES FOR CLASSES!!!!

teams__loading-container or something like this. if you are using such common names like "loadingTop" it will may affect other nested components